apps/backend/src/common/utils/formatFixedWidth.ts

```diff
@@ -1,26 +1,3 @@
-export const transactionCode = (code: string) => {
```

Comment:
Reason for removal? 
Was this used before?

Reply:
When we first built out the parsing I had included this as a nested field, but then removed in order to keep the flat files "flat" as we had discussed handling any mapping etc after the flat files had been parsed. (This was pre-database, I think we were still using the files in cyber duck at that time ?)

Reply:
There was no criteria or specs re: the adjustment to the transaction_amt field based on these values at that time.